express/src/config/session-storage.ts

```diff
@@ -11,8 +11,10 @@ function getSessionStorage(): RequestHandler {
         console.log("Using DynamoDB session storage");
         const DynamoDBStore = connect<SessionData>(session);
         return session({
-            secret: "keyboard cat",
+            secret: sessionSecret as string,
```

Comment:
```suggestion
            secret: nonNull(sessionSecret),
```
Type assertions must be avoided as they bypass type checking. We can use the helper method to ensure the value is not null.